.travis.yml

```diff
@@ -11,21 +11,23 @@ go_import_path: github.com/tidepool-org/platform
 
 env:
   global:
-    - MONGODB=6.0.11
-    - MONGOSH=1.10.6
+    - MONGODB=7.0.2
```

Comment:
This should match the version we use in production which is 6

Reply:
Fixed